core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -201,10 +202,13 @@ namespace kagome::consensus::grandpa {
     BlockInfo best_block = round->finalizedBlock().value();
 
     auto authorities_res = authority_manager_->authorities(best_block, true);
-    if (not authorities_res.has_value()) {
-      BOOST_ASSERT(authorities_res.error().message().empty());
-    }
+    SL_CRITICAL(logger_,
```

Comment:
I don't see a conditional statement for this.

Reply:
You goddamn right! =) Thanks!